src/app/clusters/content-launch-server/content-launch-server.cpp

```diff
@@ -165,6 +166,14 @@ CHIP_ERROR ContentLauncherAttrAccess::Read(const app::ConcreteReadAttributePath
 
         return ReadFeatureFlagAttribute(endpoint, aEncoder, delegate);
     }
+    case app::Clusters::ContentLauncher::Attributes::ClusterRevision::Id: {
+        if (isDelegateNull(delegate, endpoint))
+        {
+            return CHIP_NO_ERROR;
```

Comment:
What is the expected behavior here? I believe ClusterRevision (and featureMap above for that matter) are mandatory attributes, so if the delegate is null why are we not encoding anything (like 0 for feature map) in aEncoder? Or return an error like invalid_state instead of CHIP_NO_ERROR?

Is there a code path guaranteeing that a value will be encoded over TLV in the reply?

also nitpicks: why are there extra `{}` within case elements in this switch? even the default has them which is very odd and noisy.

Reply:
Looking at the git history I can see similar code has been added at first using this PR -> [13846](https://github.com/project-chip/connectedhomeip/pull/13846)

Regarding debate should it return an error or not. I'm not hard opinionated on any of the two. In general, for apps that use media clusters the have any endpoint bigger than 2. These are allocated on the fly as clients connect to the server app and it can that client sends command to one of those invalid endpoints.

If necessary I can have them return incorrect state.

Reply:
After testing the code for endpoints where delegates were not set I see that I get error:
```
[1707156414777] [96252:1410787] [TOO] Response Failure: IM Error 0x000005C3: General error: 0xc3 (UNSUPPORTED_CLUSTER)
```
So in fact this code is not used at all. I'll remove it.